code/cmdline/cmdline.cpp

```diff
@@ -1735,6 +1737,16 @@ bool SetCmdlineParams()
 			mprintf(("Failed to parse -res parameter \"%s\". Must be in format \"<width>x<height>\".\n", Cmdline_res));
 		}
 	}
+	if(window_res_arg.found()){
+		int width = 0;
+		int height = 0;
+
+		if ( sscanf(window_res_arg.str(), "%dx%d", &width, &height) == 2 ) {
+			Cmdline_window_res.emplace(width, height);
+		} else {
+			mprintf(("Failed to parse -res parameter \"%s\". Must be in format \"<width>x<height>\".\n", Cmdline_res));
```

Comment:
Might be eligible for a warning, since this happens right at program start and the user might not look to think in the debug log for what they did wrong.

Reply:
Done!